src/components/common/context-menu/context-menu.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './context-menu.scss';
+const ContextMenu = (props) => {
+  const [visible, setVisible] = useState(false);
+
+  return (
+    <React.Fragment>
+      <button type={'button'} className={'context-menu__button'} onClick={() => setVisible(true)}>
+        <div className={'dot'}></div>
+        <div className={'dot'}></div>
+        <div className={'dot'}></div>
+      </button>
+      {visible && (
+        <ul className={'context-menu__list'}>
+          <li>
+            <button className={'context-menu__close-button'} onClick={() => setVisible(false)}>
+              X
```

Comment:
It's better to avoid using text instead of "images" in such cases because it's not really good for screenreaders